video/out/vo_gpu_next.c

```diff
@@ -887,6 +889,7 @@ static void draw_frame(struct vo *vo, struct vo_frame *frame)
             continue; // ignore already seen frames
 
         if (p->want_reset) {
+            can_interpolate = false;
```

Comment:
This seems redundant with `pl_queue_reset(p->queue)`, which already prevents interpolation from mixing in any past frames.

Reply:
It'll prevent vsync offset from being non-zero for the frame at least.

Reply:
I also don't think this is correct. I would remove it and clamp it for the first frame as we do.